### ISSUE TITLE
Document and support all @3x, @2x, 1x versions of assets

### ISSUE THREAD

**Issue by JamesVanBoxtel** [panel-attack/panel-game#215]
PA should pick the asset that will look best of the current scale.

It should transition smoothly between them.

We should document this all in the readme's

**Comment by Endaris** [comment]
We discussed this in the developer channel and the plan going forward is likely to abandon this idea and just suggest adding everything with double resolution.
As we are handling the scaling by ourselves now, there is no need to use the löve approach for autoscaling assets.

**Comment by JamesVanBoxtel** [comment]
I don't think this is fully decided yet. We need to determine if the 2x assets scaled down to a 1x resolution look good enough that we don't need to support providing 1x ones.

If so, we will need to make sure we allow specifying scale / width for all images via JSON
